code/modules/admin/callproc/callproc.dm

```diff
@@ -11,7 +11,7 @@
 	var/target = null
 	var/targetselected = 0
 
-	switch(alert("Proc owned by something?",, "Yes", "No", "Cancel"))
+	switch(alert("Proc owned by something?",, "Yes", "No", "CANCEL_N"))
```

Comment:
what's this CANCEL_N
Is it a macro? That makes sense, but then why is it inside quotes?

Reply:
DUPLICATE DEFINE. Changed because is no macro define